.github/workflows/canary-integration-test.yml

```diff
@@ -286,6 +285,18 @@ jobs:
       - name: wait for ceph to be ready
         run: tests/scripts/github-action-helper.sh wait_for_ceph_to_be_ready osd 2
 
+      - name: test toolbox-operator-image pod
+        run: |
+          tests/scripts/github-action-helper.sh create_operator_toolbox
+          toolbox=$(kubectl get pod -l app=rook-ceph-tools-rook-image -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
+          # wait a little to get the pod in running status
+          sleep 120
```

Comment:
Instead of sleeping for two minutes, can we loop and check more often (perhaps every 5s)? We need to keep the tests short if possible.